db/migrate/20180626125654_make_maintenance_zone_record.rb

```diff
@@ -0,0 +1,32 @@
+class MakeMaintenanceZoneRecord < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+    validates_presence_of   :name
+    validates_uniqueness_of :name
+
+    MAINTENANCE_ZONE_NAME = "__maintenance__".freeze
+  end
+
+  def up
+    say_with_time("Creating Maintenance Zone") do
+      zone = Zone.where(:name => Zone::MAINTENANCE_ZONE_NAME).first
```

Comment:
It just occurred to me that this is not region aware.  Please change this to `Zone.in_my_region.where(...)`, to allow migrating a global database.  You can also create a spec showing this, but will need to give each zone an id within a region.  There should be other migrations that have examples showing how to do that.